rbac/management/audit_log/model.py

```diff
@@ -95,13 +95,10 @@ def get_resource_item(self, r_type, request, *args, **kwargs):
             # TODO: update for permission related items
             return None
 
-        elif r_type == "principal":
-            principal_object = get_object_or_404(Principal, username=request.user.username, tenant=verify_tenant)
-            return principal_object.id, principal_object.username
-
     def log_create(self, request, resource):
         """Audit Log when a role or a group is created."""
-        self.principal_id, self.principal_username = self.get_resource_item("principal", request)
+        self.principal_id = request.user.user_id
```

Comment:
curious question, here the `get_resource_time` method is not called anymore ... do we still need then this part? but I dont have strong preferences ❤️ https://github.com/RedHatInsights/insights-rbac/blob/34cb1b4d3398d254a599675eb83f84cf40d36a79/rbac/management/audit_log/model.py#L98-L100